src/__tests_/App.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render, fireEvent, cleanup } from 'test-utils';
+import '@testing-library/jest-dom';
```

Comment:
i don't think this is necessary as it's imported in `jest.config.js`

Reply:
Correct, works without it